### PR TITLE
Kriselda/age notification

### PR DESCRIPTION
**Pull request by kbonifacio**
created age notification component and routed it to  "/"
Changed home route to "/home" to allow the age verification buttons (nav links) to go to home or an <Underaged /> (simple page saying something along the lines of "Thank you for your interest, see you in a few years" or " Please find an adult of age to accompany you")

Need help with making the navbar disappear for this page only.

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -2,14 +2,23 @@ import React, { useState, useEffect } from 'react';
 import { Routes, Route, useNavigate, NavLink } from "react-router-dom";
 import './App.css';
 import carticon from '../images/carticon.png';
+import navName from '../images/navName.png'
+
+import {
```
Comment (Felix-Cadiz) [inline src/components/App.js:7]: We can get rid of this starter code.

File: src/components/HomeAgeVerification.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
```
Comment (Felix-Cadiz) [inline src/components/HomeAgeVerification.js:1]: Will the answer be stored in local storage? Will the user be able to navigate to the other pages if they refuse to answer or click no?

File: src/components/Wines.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from "react";
```
Comment (Felix-Cadiz) [inline src/components/Wines.js:1]: We could filter the products instead of creating a new component, but this is fine too.

**Review by Felix-Cadiz — COMMENTED** [review]
With the home verification:
- Will the user's answer be stored in local storage? 
- If the user clicks yes, will they be prompted with the same message if they return to the home page?
- What prevents a user from navigating to the other pages of the site if they click no?

**Comment by kbonifacio** [comment]
Decided to add age verification into register component as a checkbox

**Pull request closed without merging**